dm/pkg/storage/utils.go

```diff
@@ -39,13 +39,13 @@ func AdjustPath(rawURL string, uniqueID string) (string, error) {
 	if u.Scheme == "" {
 		// avoid duplicate add uniqueID
 		if !strings.HasSuffix(rawURL, uniqueID) {
-			return rawURL + "." + uniqueID, nil
```

Comment:
no need to change this behaviour now

Reply:
this is because `cfg.Checkpoint.DSN` is a file and it used to be like `l.cfg.LoaderConfig.Dir/fileName`，so if we keep this format，there may be `/xxxx.xx` format in `uniqueID`.